mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -1050,14 +1056,14 @@ def _patch_points_sampling(self, points, sem_mask, replace=None):
 
             cur_coords = coords[cur_choice, :]
             cur_sem_mask = sem_mask[cur_choice]
-
-            # two criterion for patch sampling, adopted from PointNet++
-            # points within selected patch shoule be scattered separately
+            point_idxs = np.where(cur_choice)[0]
             mask = np.sum(
-                (cur_coords >= (cur_min - 0.01)) * (cur_coords <=
-                                                    (cur_max + 0.01)),
+                (cur_coords >= (cur_min - self.eps)) * (cur_coords <=
```

Comment:
This won't introduce BC-breaking because `0.01` is set as the default value of `eps`